helper-cli/src/main/kotlin/commands/ExportLicenseFindingCurationsCommand.kt

```diff
@@ -19,63 +19,58 @@
-)
-internal class ExportLicenseFindingCurationsCommand : CommandWithHelp() {
-    @Parameter(
-        names = ["--license-finding-curations-file"],
-        required = true,
-        order = PARAMETER_ORDER_MANDATORY,
-        description = "The output license finding curations file."
-    )
-    private lateinit var licenseFindingCurationsFile: File
+internal class ExportLicenseFindingCurationsCommand : CliktCommand(
+    name = "export-license-finding-curations",
+    help = "Export the license finding curations to a file which maps repository URLs to the license finding " +
+            "curations for the respective repository."
+) {
+    private val licenseFindingCurationsFile by option(
+        "--license-finding-curations-file",
+        help = "The output license finding curations file."
+    ).convert { it.expandTilde() }
+        .file(mustExist = false, canBeFile = true, canBeDir = false, mustBeWritable = false, mustBeReadable = false)
+        .required()
 
-    @Parameter(
-        names = ["--ort-result-file"],
-        required = true,
-        order = PARAMETER_ORDER_MANDATORY,
-        description = "The input ORT file from which the license finding curations are to be read."
-    )
-    private lateinit var ortResultFile: File
+    private val ortResultFile by option(
+        "--ort-result-file",
+        help = "The input ORT file from which the license finding curations are to be read."
+    ).convert { it.expandTilde() }
+        .file(mustExist = true, canBeFile = true, canBeDir = false, mustBeWritable = false, mustBeReadable = true)
+        .required()
 
-    @Parameter(
-        names = ["--repository-configuration-file"],
-        required = false,
-        order = PARAMETER_ORDER_OPTIONAL,
-        description = "Override the repository configuration contained in the given input ORT file."
-    )
-    private lateinit var repositoryConfigurationFile: File
+    private val repositoryConfigurationFile by option(
+        "--repository-configuration-file",
+        help = "Override the repository configuration contained in the given input ORT file."
+    ).convert { it.expandTilde() }
+        .file(mustExist = true, canBeFile = true, canBeDir = false, mustBeWritable = false, mustBeReadable = true)
+        .required()
 
-    @Parameter(
-        names = ["--update-only-existing"],
-        required = false,
-        order = PARAMETER_ORDER_OPTIONAL,
-        description = "If enabled, only entries are imported for which an entry already exists which differs " +
-                "only in terms of its concluded license, comment or reason."
-    )
-    private var updateOnlyExisting = false
+    private val updateOnlyExisting by option(
+        "--update-only-existing",
+        help = "If enabled, only entries are imported for which an entry already exists which differs only in terms " +
+                "of its concluded license, comment or reason."
+    ).convert { it.expandTilde() }
```

Comment:
This should only be done for `File` options.

Reply:
Done.